### PR TITLE
Update README files for v3

### PR DESCRIPTION
**Pull request by sappenin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,82 +1,266 @@
...
+
+Xrpl4j is structured as a Maven multi-module project, with the following modules:
+
+- **xrpl4j-core
+  **: [![javadoc](https://javadoc.io/badge2/org.xrpl/xrpl4j-binary-codec/javadoc.svg?color=blue)](https://javadoc.io/doc/org.xrpl/xrpl4j-binary-codec)
+    - Provides Java objects which model XRP Ledger objects, as well as request parameters and response results for the
+      rippled websocket and JSON RPC APIs
+    - Also provides a Jackson `ObjectMapper` and JSON bindings which can be used to serialize and deserialize to and
+      from JSON
+    - Serializes the JSON representation of XRPL Transactions to the canonical binary format of the XRP Ledger
```
Comment (nkramer44) [inline README.md:228]: Should we provide links to READMEs for each package?

File: xrpl4j-core/src/main/java/org/xrpl/xrpl4j/codec/binary/README.md [hunk 2]
```diff
@@ -1,46 +1,40 @@
...
+  "  \"DestinationTag\" : 2,\n"+
+  "  \"TransactionType\" : \"Payment\"\n"+
+  "}";
+  System.out.println("JSON: \n"+paymentJson);
+
+  String binary=binaryCodec.encode(paymentJson);
+  System.out.println("Binary hex: "+binary);
+
+  String decodedJson=binaryCodec.decode(binary);
+  System.out.println("Decoded JSON: \n"+decodedJson);
```
Comment (nkramer44) [inline xrpl4j-core/src/main/java/org/xrpl/xrpl4j/codec/binary/README.md:33]: Formatting seemed better before, no?

**Review by nkramer44 — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,82 +1,266 @@
...
+# xrpl4j: A 100% Java SDK for the XRP Ledger
+
 [![codecov][codecov-image]][codecov-url]
 [![issues][github-issues-image]][github-issues-url]
 [![javadoc](https://javadoc.io/badge2/org.xrpl/xrpl4j-parent/javadoc.svg?color=blue)](https://javadoc.io/doc/org.xrpl/xrpl4j-parent)
 
-A pure Java implementation of the core functionality necessary to interact with the XRP Ledger.  While this library does not provide a network client, it does support the difficult tasks of XRPL serialization and transaction signing, and provides useful Java bindings for XRP Ledger objects and rippled request/response objects.  
+This project is a pure Java implementation of an SDK that works with the XRP Ledger. This library supports XRPL
+transaction serialization and signing, provides useful Java bindings for XRP Ledger objects and rippled request/response
+objects, and also provides a JSON-RPC client for interacting with XRPL nodes.
```
[the remark this answers is not in the document]
Comment (sappenin) [reply README.md:9]: Added via 85c057027265e030d95ad767230ef3e3bfd190e4

File: README.md [hunk 4]
```diff
@@ -1,82 +1,266 @@
...
+  .sequence(UnsignedInteger.ONE)
+  .amount(XrpCurrencyAmount.ofDrops(10000))
+  .destination(Address.of("rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW"))
+  .destinationTag(UnsignedInteger.valueOf(23480))
+  .cancelAfter(UnsignedLong.valueOf(533257958))
+  .finishAfter(UnsignedLong.valueOf(533171558))
+  .condition(CryptoConditionReader.readCondition(
+  BaseEncoding.base16()
+  .decode("A0258020E3B0C44298FC1C149AFBF4C8996FB92427AE41E4649B934CA495991B7852B855810100"))
+  )
```
[the remark this answers is not in the document]
Comment (sappenin) [reply README.md:91]: Fixed via fe796da074cf7f365763ec3b5a52af1a84e3daca

File: README.md [hunk 5]
```diff
@@ -1,82 +1,266 @@
...
+Jackson `ObjectMapper`, which can be instantiated
+using [`ObjectMapperFactory`](./xrpl4j-core/src/main/java/org/xrpl/xrpl4j/model/jackson/ObjectMapperFactory.java).
+
+Using the `EscrowCreate` object created above, it is then possible to use the supplied `ObjectMapper` to serialize to
+JSON like this:
+
+```java
+ObjectMapper objectMapper=ObjectMapperFactory.create();
+  String json=objectMapper.writerWithDefaultPrettyPrinter().writeValueAsString(escrowCreate);
+  System.out.println(json);
```
[the remark this answers is not in the document]
Comment (sappenin) [reply README.md:106]: Fixed via 0aa86664a62391b5765ddd8f6cb152aa7fd45bb1

File: README.md [hunk 6]
```diff
@@ -1,82 +1,266 @@
...
+import org.xrpl.xrpl4j.crypto.core.keys.PrivateKey;
+import org.xrpl.xrpl4j.crypto.core.keys.PublicKey;
+import org.xrpl.xrpl4j.model.transactions.Address;
+  
+...
+
+Seed seed=Seed.ed255519Seed(); // <-- Generates a random seed.
+PrivateKey privateKey=seed.derivePrivateKey(); // <-- Derive a private key from the seed.
+PublicKey publicKey=privateKey.derivePublicKey(); // <-- Derive a public key from the private key.
+Address address=publicKey.deriveAddress(); // <-- Derive an address from the public key.
```
[the remark this answers is not in the document]
Comment (sappenin) [reply README.md:156]: Fixed via 3c49d3717460cdff0a4ba9e206575701e18c099b

File: README.md [hunk 7]
```diff
@@ -1,82 +1,266 @@
...
+Seed senderSeed = Seed.ed255519Seed();
+PrivateKey senderPrivateKey=senderSeed.derivePrivateKey();
+PublicKey senderPublicKey=senderPrivateKey.derivePublicKey();
+Address senderAddress=senderPublicKey.deriveAddress();
+
+// Receiver (using secp256k1 key)
+Address receiverAddress=Address.of("r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59");
+
+// Construct a Payment
+Payment payment=...; // See V3 ITs for examples.
```
[the remark this answers is not in the document]
Comment (sappenin) [reply README.md:209]: Fixed via 3c49d3717460cdff0a4ba9e206575701e18c099b

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (sappenin) [reply README.md:106]: Agreed. Updated via 7d1a8e18820502907bfc9cf2ad86697976937f8a

File: README.md (hunk 1 above)
Comment (sappenin) [reply README.md:227]: Hmmm...good question. Do you envision this going into the project's main README then?

File: xrpl4j-core/src/main/java/org/xrpl/xrpl4j/codec/binary/README.md (hunk 2 above)
Comment (sappenin) [reply xrpl4j-core/src/main/java/org/xrpl/xrpl4j/codec/binary/README.md:33]: Agreed. Fixed.

File: README.md (hunk 1 above)
Comment (sappenin) [reply README.md]: Fixed via cf2592bf80efa5a44b8583e605e302701783044c

File: README.md [hunk 8]
```diff
@@ -1,82 +1,264 @@
...
 ## Development
+
+### Project Structure
+
+Xrpl4j is structured as a Maven multi-module project, with the following modules:
+
+- **xrpl4j-core**: [![javadoc](https://javadoc.io/badge2/org.xrpl/xrpl4j-binary-codec/javadoc.svg?color=blue)](https://javadoc.io/doc/org.xrpl/xrpl4j-binary-codec)
+    - Provides core primitives like seeds, public/private keys definitions (supports secp256k1 and ed25519 key types 
+      and signing algorithms), signature interfaces, etc. Also provides Java objects which model XRP Ledger objects, 
+      as well as request parameters and response results for the `rippled` websocket and JSON RPC APIs.
```
Comment (nkramer44) [inline README.md:230]:
```suggestion
    - Provides core primitives like seeds, public/private keys definitions (supports secp256k1 and ed25519 key types 
      and signing algorithms), signature interfaces, address and binary codecs etc. Also provides Java objects which model XRP Ledger objects, 
      as well as request parameters and response results for the `rippled` websocket and JSON RPC APIs.
```

**Review by nkramer44 — APPROVED** [review]
Just one small thing. Otherwise looks good!

**Pull request merged**